File: genAlgo/banana/runGenAlgo_banana.py
from banana import genAlgo_banana
import tellurium as te
from banana import models_banana

runID = models_banana.makeFolder('data/',date = True)


# to run Michaelis-Menten model
populationFolder = models_banana.generateModelFiles(100, groundTruthModel_string=models_banana.groundTruth_e, parameters=models_banana.K_LIST, folderName=runID)
genAlgo_banana.runGeneticAlgorithm(populationFolder, groundTruth=models_banana.groundTruth_e, parameters=models_banana.K_LIST, survivorRatio=(10,90), lastGeneration=300, tolerance=0.001, runID=runID)

File: genAlgoJL/banana/runGenAlgo_banana.jl
# make sure to pwd to genAlgo\\banana before running this code

include("models_banana.jl")
include("evaluate_banana.jl")
include("genAlgo_banana.jl")

using Main.Banana_Models
using Main.Banana_GenAlgo

using PyCall
te = pyimport("tellurium")

# cd("genAlgo\\banana")

runID = Main.Banana_Models.makeFolder("data/", true)

# to run Michaelis-Menten model
populationFolder = Main.Banana_Models.generateModelFiles(100, runID)
# runID = "data/20211108_4"
# populationFolder = "data/20211108_4/banana_population_1"

Main.Banana_GenAlgo.runGeneticAlgorithm(populationFolder, runID)


